src/ruby/embed.js

```diff
@@ -36,6 +36,12 @@ function replaceNewlines(doc) {
   );
 }
 
+function removeBreakParent(doc) {
+  return mapDoc(doc, (currentDoc) =>
+    currentDoc.type === "break-parent" ? [] : currentDoc
```

Comment:
@mlauter I'm not quite sure how, but it does seem to work.  My guess is that `mapDoc` doesn't recurse deeply and therefore only removes `break-parent` commands from the outermost elements of the body.  I'm adding a test case with your example code because it seems like a good thing to ensure.

Reply:
...although, that example code turns out to break the keep-parent-indentation part of this, so back to the drawing board somewhat 😭

Reply:
OK, pushed another version of this that I don't understand how it works.  @jennceng and I spent awhile doing some trial and error, and it looks like we don't need to remove `break-parent` commands if we insert a `dedent` command.

Again, I have no idea why this works, but it seems to work and doesn't break any of the other tests.  Lolsob.